<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Fieldwork Model Source Step</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="idLabel">
        <property name="text">
         <string>Identifier:  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="gfLabel">
        <property name="text">
         <string>GF Filename:  </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ensLabel">
        <property name="text">
         <string>Ensemble Filename:  </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="meshLabel">
        <property name="text">
         <string>Mesh Filename:  </string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path:  </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="gfLocLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="gfLocButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="ensLocLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="ensLocButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="meshLocLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="meshLocButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="pathLocLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pathLocButton">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>idLineEdit</tabstop>
  <tabstop>gfLocLineEdit</tabstop>
  <tabstop>gfLocButton</tabstop>
  <tabstop>ensLocLineEdit</tabstop>
  <tabstop>ensLocButton</tabstop>
  <tabstop>meshLocLineEdit</tabstop>
  <tabstop>meshLocButton</tabstop>
  <tabstop>pathLocLineEdit</tabstop>
  <tabstop>pathLocButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
